apps/dot-voting/app/components/VotingOptions.js

```diff
@@ -13,6 +13,7 @@ class VotingOptions extends React.Component {
     voteWeights: [],
     // animationDelay can also be a number to disable the random delay
     animationDelay: { min: ANIM_DELAY_MIN, max: ANIM_DELAY_MAX },
+    displayYouBadge: false,
```

Comment:
I would say that we don't need to add false as default prop, since undefined will be evaluated also as false at line 61